website/content/docs/commands/volume/create.mdx

```diff
@@ -0,0 +1,157 @@
+  to the volume.
+
+- `name` `(string: <required>)` - The display name of the volume.
+
+- `type` `(string: <required>)` - The type of volume. Currently only `"csi"`
+  is supported.
+
+- `external_id` `(string: <required>)` - The ID of the physical volume from
+  the storage provider. For example, the volume ID of an AWS EBS volume or
+  Digital Ocean volume.
+
+- `plugin_id` `(string: <required>)` - The ID of the [CSI plugin][csi_plugin]
+  that manages this volume.
+
+- `snapshot_id` `(string: <optional>)` - If the storage provider supports
+  snapshots, the external ID of the snapshot to restore when creating this
+  volume. If omitted, the volume will be created from scratch. The
+  `snapshot_id` cannot be set if the `clone_id` field is set.
+
+- `clone_id` `(string: <optional>)` - If the storage provider supports
+  cloning, the external ID of the volume to clone when creating this
+  volume. If omitted, the volume will be created from scratch. The
+  `clone_id` cannot be set if the `snapshot_id` field is set.
+
+- `capacity_min` `(string: <optional>)` - Option for setting the capacity. The
+  volume must be at least this large, in bytes. The storage provider may
+  return a volume that is larger than this value. Accepts human-friendly
+  suffixes such as `"100GiB"`. This field may not be supported by all
+  storage providers.
+
+- `capacity_max` `(string: <optional>)` - Option for setting the capacity. The
+  volume must be no more than this large, in bytes. The storage provider may
+  return a volume that is smaller than this value. Accepts human-friendly
+  suffixes such as `"100GiB"`. This field may not be supported by all
+  storage providers.
+
+- `capability` `(optional)` - Option for validating the capbility of a
+  volume. You may provide multiple `capability` blocks, each of which must
+  have the following fields:
+
+  - `access_mode` `(string: <required>)` - Defines whether a volume should be
+    available concurrently. Can be one of `"single-node-reader-only"`,
+    `"single-node-writer"`, `"multi-node-reader-only"`,
+    `"multi-node-single-writer"`, or `"multi-node-multi-writer"`. Most CSI
+    plugins support only single-node modes. Consult the documentation of the
+    storage provider and CSI plugin.
+
+  - `attachment_mode` `(string: <required>)` - The storage API that will be used
+    by the volume. Most storage providers will support `"file-system"`, to mount
+    volumes using the CSI filesystem API. Some storage providers will support
+    `"block-device"`, which will mount the volume with the CSI block device API
+    within the container.
+
+- `mount_options` - Options for mounting `file-system` volumes that don't
+  already have a pre-formatted file system. Consult the documentation for your
+  storage provider and CSI plugin as to whether these options are required or
+  necessary.
+
+  - `fs_type`: file system type (ex. `"ext4"`)
+  - `mount_flags`: the flags passed to `mount` (ex. `"ro,noatime"`)
```

Comment:
should these get type hits for string and array? the ex kind of makes it look like it is a comma separated string, but in other examples it looks like it should be an array

Reply:
Good catch! Will fix.